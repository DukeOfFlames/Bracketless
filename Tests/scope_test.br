START

°global = "I am a global variable"

{
    °a = 1
    {
        °a = 2
        {
            °a = 3
            #drucke{a}
        }
        a = 4
        #drucke{a}
    }
    a = 5
    #drucke{a}
}

{
    °b = 1
    {
        b = 2
        {
            b = 3
            #drucke{b}
        }
        b = 4
        #drucke{b}
    }
    b = 5
    #drucke{b}
}

{
    °l = {1, 2, 3}
    #drucke{l}
}

END