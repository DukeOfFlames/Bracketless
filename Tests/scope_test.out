[Integer:
  3]
[Integer:
  4]
[Integer:
  5]
[Integer:
  3]
[Integer:
  4]
[Integer:
  5]
[List:
  [
    Integer:
      1
    Integer:
      2
    Integer:
      3
  ]]
